functions/Copy-SqlMaintenancePlan.ps1

```diff
@@ -0,0 +1,237 @@
+
+If the maintenance plan already exist on the destination, it will be skipped unless -Force is used.
+
+.PARAMETER Source
+Source SQL Server.You must have sysadmin access and server version must be SQL Server version 2000 or greater.
+
+.PARAMETER Destination
+Destination Sql Server. You must have sysadmin access and server version must be SQL Server version 2000 or greater.
+
+.PARAMETER SourceSqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
+
+$scred = Get-Credential, then pass $scred object to the -SourceSqlCredential parameter.
+
+Windows Authentication will be used if DestinationSqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.
+To connect as a different Windows user, run PowerShell as that user.
+
+.PARAMETER DestinationSqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
+
+$dcred = Get-Credential, then pass this $dcred to the -DestinationSqlCredential parameter.
+
+Windows Authentication will be used if DestinationSqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.
+
+To connect as a different Windows user, run PowerShell as that user.
+
+.NOTES 
+Author: Shawn Melton (@wsmelton)
+Requires: sysadmin access on SQL Servers
+
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/Copy-SqlMaintenancePlan
+
+.EXAMPLE
+Copy-SqlMaintenancePlan -Source sqlserver2014a -Destination sqlcluster
+Copies all maintenance plans from sqlserver2014a to sqlcluster, using Windows credentials. If maintenance plans exist with the same name on sqlcluster, they will be skipped.
+
+.EXAMPLE   
+Copy-SqlMaintenancePlan -Source sqlserver2014a -Destination sqlcluster -Plan MyBackup -SqlCredential $cred -Force
+
+Copies a single maintenance plan, the MyBackup, from sqlserver2014a to sqlcluster, using SQL credentials for sqlserver2014a and Windows credentials for sqlcluster. If a plan with the same name exist on sqlcluster, it will be removed and recreated because -Force was used.
+
+.EXAMPLE   
+Copy-SqlMaintenancePlan -Source sqlserver2014a -Destination sqlcluster -WhatIf -Force
```

Comment:
@ctrlbold ok, think latest commit should fix all of this...one stupid variable change after looking at it for 3 days. 🤕